syft/src/main/java/org/openmined/syft/datasource/JobLocalDataSource.kt

```diff
@@ -11,7 +12,7 @@ import java.io.InputStream
 internal const val DIFF_SCRIPT_NAME = "diff_script.pt"
 private const val TAG = "JobLocalDataSource"
 
-internal class JobLocalDataSource {
+internal class JobLocalDataSource(@VisibleForTesting internal val dataModel: JobDataModel) {
```

Comment:
no need for internal specification here I guess. Also visible for testing is also default